src/plugins/cosmos/components/StakingInput/StakingInput.tsx

```diff
@@ -19,7 +19,8 @@ type StakingInputProps = {
 const cryptoInputValidation = {
   required: true,
   validate: {
-    validateCryptoAmount: (_: string) => {
+    validateCryptoAmount: (cryptoAmount: string) => {
+      if (bnOrZero(cryptoAmount).isZero()) return false
       // TODO: Implement when we have cosmos/osmosis balance data
```

Comment:
is this comment still valid? we def do have cosmos balance data?

Reply:
Nice catch, comment's gone as this is pretty much the implementation we needed here